windows_docker/src/win_docker.py

```diff
@@ -0,0 +1,120 @@
+from checks import AgentCheck
+from datetime import tzinfo, timedelta
+import datetime
+import requests
+import json
+import calendar
+
+class UTC(tzinfo):
+  def utcoffset(self, dt):
+    return timedelta(0)
+  def tzname(self, dt):
+    return "UTC"
+  def dst(self, dt):
+    return timedelta(0)
+
+class WinDocker(AgentCheck):
+	def check(self, instance):
+		container_full_ids = []
+		container_ids = []
+		container_urls = []
+		id_tag = ""
+		image_tag = ""
+		state_tag = ""
+		status_tag = ""
+		name_tag = ""
+		net_tmpl = "docker.windows.networks.{}"
+		blkio_stats_tmpl = "docker.windows.blkio_stats.{}"
+		storage_stats_tmpl = "docker.windows.storage_stats.{}"
+		cpu_stats_tmpl = "docker.windows.cpu_stats.cpu_usage.{}"
+		throttling_data_tmpl = "docker.windows.cpu_stats.throttling_data.{}"
+		precpu_stats_cpu_usage_tmpl = "docker.windows.precpu_stats.cpu_usage.{}"
+		precpu_stats_throttling_data_tmpl = "docker.windows.precpu_stats.throttling_data.{}"
+		memory_stats_tmpl = "docker.windows.memory_stats.{}"
+
+		container_url="http://localhost:2375/containers/json"
+		containers_response = requests.get(container_url, timeout=1).json()
+
+
+		for value in containers_response:
+			container_full_ids.append(value["Id"])
+			container_count = len(containers_response)
+			
+			if value:
+				id_tag = "sources:docker_win"
+				image_tag = "docker_image:" + value["Image"]
+				state_tag = "state:" + value["State"]
+				status_tag = "status:" + value["Status"]
+				name_tag = "container_name:" + value["Names"][0]
+				self.gauge("docker.windows.created", value["Created"], tags=[id_tag, image_tag, name_tag, status_tag, state_tag ])
+	# running containers:
+		self.gauge("docker.windows.containers.running", container_count, tags=[id_tag, image_tag, name_tag, status_tag, state_tag])
```

Comment:
Generally speaking it would be useful to retrieve the same metrics (and keep the same name) as on Linux, so that users who run hybrid environments can display values of both linux and windows side by side, splitting them only with tags.

Reply:
Gotcha, okay - I can remove all the `.windows.` from the metrics and just add a tag for os: windows, does that sound okay - or is there a specific tag that you would like me to reference?